dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -312,6 +312,7 @@ def jquery_is_element_open(selector)
 When /^I press "([^"]*)"(?: to load a new (page|tab))?$/ do |button, load|
   wait_short_until do
     @button = @browser.find_element(id: button)
+    @button&.displayed?
```

Comment:
When doing jQuery animations you sometimes have the element added a quarter second before its clickable